### DISCUSSION TITLE
How to change the default port number of Jobrunr

### DISCUSSION THREAD

**Discussion by talk2morris** [jobrunr/jobrunr#676] [General]
I am curious to know if it is possible to change the default port number of Jobrunr, that would help to spin up several instances in the same machine.

**Comment by rdehuyss** [comment]
It's all in the docs 😄 .

See https://www.jobrunr.io/en/documentation/configuration/spring/ and search for port.

**Comment by talk2morris** [comment]
thanks, I don't know how I missed that.
